<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAValueTableWidget</class>
 <widget class="QWidget" name="SAValueTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Value Viewer Tab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SAValueDataTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionToLinerData">
   <property name="text">
    <string>导为线性数组</string>
   </property>
   <property name="toolTip">
    <string>把选择的一个或多个列数据导为线性数组</string>
   </property>
  </action>
  <action name="actionToPointFVectorData">
   <property name="text">
    <string>导为点数组</string>
   </property>
   <property name="toolTip">
    <string>把选择的两行或两列转为点序列</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAValueDataTableView</class>
   <extends>QTableView</extends>
   <header>SAValueDataTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
